md/first-steps-after-setup.md

```diff
@@ -12,11 +12,43 @@ In the URL field of your web browser, enter the local host address, e.g. `http:/
+
+#### Use HTTPS for all communications
+HTTPS helps with the protection of web application data from unauthorized disclosure and modification when it is transmitted between clients (web browsers) and the web application server. Session cookies, authentication credentials and any sensitive data  are considered as application data. Bonita already supports HTTPS functionality. To better protect your deployed application, it is highly recommend to enable HTTPS for all communication towards the web server. Please refer to [OWASP TLS Certificates](https://www.owasp.org/index.php/Transport_Layer_Protection_Cheat_Sheet#Server_Certificate) and [Bonita's SSL Documentation](ssl.md) to securely implement this functionality for your application.
+
+#### Enforce strong secure password policies to Bonita users
+A key security concern when using passwords for authentication is password strength. A "strong" password policy makes it difficult or even improbable for one to guess the password through either manual or automated means. So if you are using Bonita's default authentication mechanism, it is highly recommended that you apply the password policies and enforce the users to choose strong and secure passwords. Such functionality is already provided by Bonita (See [Strong Password Policies](enforce-password-policy.md)). It is recommended to activate this option in the deployed application to better protect your application and users.
+
+#### Choose a strong password for your database
+The database is at the heart of your application. It is where all entreprise data (sensitive or non-sensitive) is stored. As a result, the database username and password are of great value to the business and they need to be protected. It is highly recommended to choose strong, random and long passwords, since they make it difficult or even improbable for one to guess the password through either manual or automated means. 
+
+#### Choose a strong password for your connectors
+In Bonita, connectors enable connections to your local information system or online services. The authentication credentials used for these connections are of great value to the business and as a result, they need to be protected. It is highly recommended to choose strong, random and long passwords, as explained above, to makes it difficult or even improbable for one to guess the password. 
+
+#### Prohibit or limit the usage of Java Client
+The Java Client is a tool that can be downloaded and installed for configuration purposes by the administrators. If such tool is of no use for your application, it is highly recommended to prohibit the usage of Java Client in your deployment environment. Otherwise, its usage must be limited to administrators using a local connection or authorized IP addresses.
```

Comment:
What do you mean with Java Client? How can you prohibit its usage?
Don't you rather mean to deactivate the Engine HTTP API?

Reply:
Thanks for your comment. I didn't know that  Engine HTTP API can be deactivated in configuration files. I'll change and reformulate this part.

Reply:
See https://documentation.bonitasoft.com/bonita/7.3/configure-client-of-bonita-bpm-engine#toc5 for server side configuration